_includes/concepts.md

```diff
@@ -55,9 +55,10 @@ Nicknames are non-empty strings with the following restrictions:
 - They MUST NOT contain any of the following characters: space `(' ', 0x20)`, comma `(',', 0x2C)`, asterisk `('*', 0x2A)`, question mark `('?', 0x3F)`, exclamation mark `('!', 0x21)`, at sign `('@', 0x40)`.
 - They MUST NOT start with any of the following characters: dollar `('$', 0x24)`, colon `(':', 0x3A)`.
 - They MUST NOT start with a character listed as a [channel type](#channel-types) prefix.
+- They MUST NOT start with a character listed as a [channel membership prefix](#channel-membership-prefixes), or prefix listed in the IRCv3 [`multi-prefix` Extension](https://ircv3.net/specs/extensions/multi-prefix).
 - They SHOULD NOT contain any dot character `('.', 0x2E)`.
```

Comment:
This line should probably be merged into "They MUST NOT contain any of the following characters..."